Backend/build.gradle

```
@@ -0,0 +1,105 @@
+buildscript {
+    ext {
+        queryDslVersion = "5.0.0"
+        snippetsDir = file("build/generated-snippets")
+    }
+}
+
+plugins {
+    id 'org.springframework.boot' version '2.6.0'
+    id 'io.spring.dependency-management' version '1.0.11.RELEASE'
+    id 'java'
+    id 'com.ewerk.gradle.plugins.querydsl' version '1.0.10'
+    id "org.asciidoctor.jvm.convert" version "3.3.2"
+}
+
+group = 'com.team33'
+version = '0.0.1-SNAPSHOT'
+sourceCompatibility = '11'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation('org.springframework.boot:spring-boot-starter-web')
+    implementation('org.springframework.boot:spring-boot-starter')
+    implementation("org.springframework.boot:spring-boot-starter-data-jpa")
+
+    implementation('org.projectlombok:lombok')
+
+    implementation("com.querydsl:querydsl-jpa:${queryDslVersion}")
+    implementation("com.querydsl:querydsl-apt:${queryDslVersion}")
+
+    implementation(group: 'io.jsonwebtoken', name: 'jjwt-api', version: '0.11.2')
+    implementation(group: 'io.jsonwebtoken', name: 'jjwt-impl', version: '0.11.2')
+    implementation(group: 'io.jsonwebtoken', name: 'jjwt-jackson', version: '0.11.2')
```

이 의존성들은 어디에 사용 될 예정인가요??

---

`jjwt` 라이브러리는 인증/세션 처리를 위한 jwt 토큰 발급/검증 용으로 사용되는 라이브러리 입니다.
이 라이브러리를 사용하기 위해서는 api, imple, jackson 세 개의 의존성이 모두 필요합니다.
[공식 Repo](https://github.com/jwtk/jjwt#gradle)

- `jjwt-api` : 호환성을 위한 인터페이스에 해당함
- `jjwt-impl` : `jjwt-api` 의 실제 구현에 해당하는 라이브러리
- `jjwt-jackson` : `jjwt-impl` 이 사용하는 라이브러리

그래서 현재는 모두 `implementation` 이지만, 이를 `compileOnly`, `runtimeOnly` 로 분리할 예정입니다.

```suggestion
    compileOnly(group: 'io.jsonwebtoken', name: 'jjwt-api', version: '0.11.2')
    runtimeOnly(group: 'io.jsonwebtoken', name: 'jjwt-impl', version: '0.11.2')
    runtimeOnly(group: 'io.jsonwebtoken', name: 'jjwt-jackson', version: '0.11.2')
```